hawkular-alerts-api/src/main/java/org/hawkular/alerts/api/model/trigger/Trigger.java

```diff
@@ -58,154 +60,115 @@
 
     private static final long serialVersionUID = 1L;
 
-    @ApiModelProperty(value = "Tenant id owner of this trigger.",
-            position = 0,
-            required = true,
-            allowableValues = "Tenant is overwritten from Hawkular-Tenant HTTP header parameter request")
+    @ApiModelProperty(value = "Tenant id owner of this trigger.", position = 0, required = true, allowableValues = "Tenant is overwritten from Hawkular-Tenant HTTP header parameter request")
```

Comment:
Why these changes of formatting ?
Please, let's respect formats.
IDE should be put in mode "don't modify formats automatically", in this case is painful, as these document annotations are hard to read without proper formatting.
I'm tired to battle about this.

I think the rule about "ok, if someone defines a format first, let's respect it" is enough fair.